src/meta/src/lib.rs

```diff
@@ -269,8 +269,7 @@ pub fn start(opts: MetaNodeOpts) -> Pin<Box<dyn Future<Output = ()> + Send>> {
                 tokio::select! {
                     _ = &mut handle => {
                         tracing::info!("receive leader lost signal");
```

Comment:
Should this not be `tracing::error` or higher? We kill the node here.

Reply:
Before triggering this log, the election client would have already output a lot of logs about election. It's sufficient to just mark it here.